k8s/upgrades/0.9.0-1.0.0/jiva/jiva_volume_upgrade.sh

```diff
@@ -208,17 +221,101 @@ fi
 # #### PATCH TARGET SERVICE ####
 if [[ "$controller_svc_version" != "$target_upgrade_version" ]]; then
     echo "Upgrading Target Service to $target_upgrade_version"
-    kubectl patch service --namespace "$ns" "$c_svc_name" -p "$(cat jiva-target-svc-patch.json)"
-    rc=$?; if [ $rc -ne 0 ]; then echo "Failed to patch the service $c_svc_name \
-            | Exit code: $rc"; exit; fi
+    kubectl patch service --namespace "$ns" "$c_svc_name" \
+        -p "$(cat jiva-target-svc-patch.json)"
+    rc=$?; if [ $rc -ne 0 ]; then 
+    echo "Failed to patch the service $c_svc_name | Exit code: $rc"; exit; 
+fi
 else
     echo "Controller service $c_svc_name is already at $target_upgrade_version"
 fi
 
-echo "Clearing temporary files"
-rm jiva-replica-patch.json
-rm jiva-target-patch.json
-rm jiva-target-svc-patch.json
+#Verifying the upgraded deployment versions
+controller_version=$(kubectl get deployment "$c_deploy_name" -n "$ns" \
+        -o jsonpath='{.metadata.labels.openebs\.io/version}')
+if [ "$controller_version" != "$target_upgrade_version" ]; then
+    echo "Failed to upgrade Target deployment $c_deploy_name labels";exit 1;
```

Comment:
`T` should be `t`
similar comment in other error logs also `Replica` as `replica` in error message

Reply:
Target to target
Controller to controller
Service to service

Reply:
Also all these verification steps related error message should be like:
`Failed in validating <so and so>` (or) `Validation failed for <so and so>`
Otherwise, with the kind of errors in PR, we don't know whether upgrade failed or validation failed.